beginners-yoga-course.html

```diff
@@ -0,0 +1,79 @@
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Prema Healing Cleveland</title>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Signika:300">
+  <link rel="stylesheet" href="main.css">
+  <style type="text/css"></style>
+</head>
+<body>
+<div class="pageWrap">
+
+<!-- HEADER -->
+<div class="gridCell--row gridCell--padded gridCell--row--justifyCenter">
+  <header class="pageHeader">
+    <h1 class="headline pageHeader__headline align--center">
+      <span class="headline__line pageHeader__primary inverted">Prema Healing</span>
+      <span class="headline__line pageHeader__secondary">Cleveland</span>
+    </h1>
+  </header>
+</div>
```

Comment:
It's pretty common for the site "logo" to link back to the homepage. Now that the site has multiple pages, you might consider doing that.